LICENSE-binary

```diff
@@ -247,7 +247,6 @@ org.apache.thrift:libfb303
 org.apache.thrift:libthrift
 org.webjars:swagger-ui
 org.yaml:snakeyaml
-log4j:log4j
```

Comment:
it still on our `depenencyList` can we also remove it there?

Reply:
```
    <dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>org.apache.kyuubi</groupId>
                <artifactId>kyuubi-common_${scala.binary.version}</artifactId>
                <version>${project.version}</version>
                <exclusions>
                    <exclusion>
                        <groupId>log4j</groupId>
                        <artifactId>log4j</artifactId>
                    </exclusion>
                </exclusions>
            </dependency>

...
    </dependencyManagement>
```